includes/admin/customers/customers.php

```diff
@@ -597,12 +597,12 @@ function edd_customers_view( $customer = null ) {
 			<li>
 				<a href="<?php echo admin_url( 'edit.php?post_type=download&page=edd-payment-history&customer=' . $customer->id ); ?>">
 					<span class="dashicons dashicons-cart"></span>
-					<?php printf( _n( '%s Completed Sale', '%s Completed Sales', $customer->purchase_count, 'easy-digital-downloads' ), esc_html( number_format_i18n( $customer->purchase_count, 0 ) ) ); ?>
+					<?php printf( _n( '<span class="edd_purchase_count">%s</span> Completed Sale', '<span class="edd_purchase_count">%s</span> Completed Sales', $customer->purchase_count, 'easy-digital-downloads' ), esc_html( number_format_i18n( $customer->purchase_count, 0 ) ) ); ?>
 				</a>
 			</li>
 			<li>
 				<span class="dashicons dashicons-chart-area"></span>
-				<?php echo esc_html( edd_currency_filter( edd_format_amount( $customer->purchase_value ) ) ); ?> <?php esc_html_e( 'Lifetime Value', 'easy-digital-downloads' ); ?>
+				<?php echo '<span class="edd_purchase_value">' . esc_html( edd_currency_filter( edd_format_amount( $customer->purchase_value ) ) ) . '</span>'; ?> <?php esc_html_e( 'Lifetime Value', 'easy-digital-downloads' ); ?>
```

Comment:
Would be nice to update this similarly to the "Completed Sale" string for better translatability.